coralproject/talk
# [CORL-3202] Add bluesky to embed options

## What does this PR do?

Adds Bluesky embeds to the list of various other oembed types we support.

### TODO

- ~figure out why the media preview is trying to use `GifPreview` for the Bluesky embed type~
  - ~this is weird, but at least everything else is working~

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

Adds various `BlueskyMedia` types and inputs (same as say, `TwitterMedia`)

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

N/A

## How do I test this PR?

- enable Bluesky embeds in `Admin > Config > General > Embedded media`
- paste an embed url from Bluesky into your comment
- add the post to your comment
- post the comment
- see attached Bluesky oembed

## Were any tests migrated to React Testing Library?

No

## How do we deploy this PR?

- Merge into `develop`
